s3_file_field/pydantic.py

```diff
@@ -0,0 +1,16 @@
+from typing import Annotated
+
+from pydantic import PlainValidator
+
+from s3_file_field.fields import S3PlaceholderFile
+
+
+def _convert_s3ff_value(value: str) -> str:
+    file_obj = S3PlaceholderFile.from_field(value)
+    if file_obj is None:
+        raise ValueError("Invalid S3 file field value")
+    return file_obj.name
```

Comment:
I'm not sure if we should ideally be returning `file_obj.name`. I recall that we have to do this for DRF as a hack to deal with how they treat files, but I'm hoping we'd be able to return the full `S3PlaceholderFile` here.

Reply:
[5697e75](https://github.com/kitware-resonant/django-s3-file-field/pull/357/commits/5697e75616f707eb75ad8019360a7ee3636694a4) shows what it takes to return the full `S3PlaceholderFile`, following the example [here](https://docs.pydantic.dev/latest/concepts/types/#handling-third-party-types). I'll leave it to you to decide if it's worth the complexity.